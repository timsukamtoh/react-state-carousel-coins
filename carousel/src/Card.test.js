import { render } from "@testing-library/react";
import Card from "./Card.js";

it("renders without crashing", function () {
  // this is a low-value test, but better than nothing
  render(<Card caption="test" src="url" currNum={1} totalNum={3} />);
});

describe("snapshot tests", function () {
  it("initial snapshot", function () {
    const { container } = render(<Card caption="test" src="url" currNum={1} totalNum={3}/>);
    expect(container).toMatchSnapshot();
  });
});