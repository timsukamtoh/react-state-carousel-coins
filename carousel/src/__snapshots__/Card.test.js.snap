// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot tests initial snapshot 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      test
    </h4>
    <img
      alt="test"
      class="Card-image"
      src="url"
    />
    <small
      class="Card-small"
    >
      Image 
      1
       of 
      3
      .
    </small>
  </div>
</div>
`;
